v3/cypress/e2e/text.spec.ts

```diff
@@ -11,20 +11,25 @@ context("Text tile", () => {
       cy.visit(url)
       cy.wait(1000) // Ensuring the page and components are fully loaded.
   })
-  it("updates text title with undo/redo", () => {
+  // this test has become flaky, skipping for now
+  // PT-#188358092
+  it.skip("updates text title with undo/redo", () => {
     const newTextTileName = "My Text"
     c.getComponentTitle(kTextTileTestId).should("have.text", textDefaultTitle)
     c.changeComponentTitle(kTextTileTestId, newTextTileName)
     c.getComponentTitle(kTextTileTestId).should("have.text", newTextTileName)
 
     cy.log("Check update text title with undo/redo")
     // Undo title change
+    cy.wait(100)
     toolbar.getUndoTool().click()
+    cy.wait(100)
     c.getComponentTitle(kTextTileTestId).should("have.text", textDefaultTitle)
 
     // Redo title change
     toolbar.getRedoTool().should("be.enabled")
     toolbar.getRedoTool().click()
+    cy.wait(100)
```

Comment:
Would it make sense to take out these added waits?

Reply:
I agree that the `wait`s should be removed unless they've been determined to be necessary (in which case there's probably a better solution anyway).

Reply:
I didn't add these in -- these changes came in when I merged in main.